File: SGUC/src/java/sguc/model/Imputado.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package sguc.model;

import java.io.Serializable;
import java.sql.Time;

/**
 *
 * @author Leo
 */
public class Imputado implements Serializable, Jsonable{
    
    String informe;
    String persona;
    /*
    String cedula;
    String nombre;
    String apellido1;
    String apellido2;
    char sexo;
    int edad;
    String nacionalidad;
    Date nacimiento;
    
    o 
    
    Persona persona;
    */
    String direccion;
    String telefono;
    String alias;
    boolean aprehendido;
    Time horaAprehension;
    boolean entendidos;
    String motivo_sinfirma;
    String rasgos;
    String vestimenta;

    public String getInforme() {
        return informe;
    }

    public void setInforme(String Informe) {
        this.informe = Informe;
    }

    public String getPersona() {
        return persona;
    }

    public void setPersona(String Persona) {
        this.persona = Persona;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public String getAlias() {
        return alias;
    }

    public void setAlias(String Alias) {
        this.alias = Alias;
    }

    public boolean isAprehendido() {
        return aprehendido;
    }

    public void setAprehendido(boolean Aprehendido) {
        this.aprehendido = Aprehendido;
    }

    public Time getHoraAprehension() {
        return horaAprehension;
    }

    public void setHoraAprehension(Time horaAprehension) {
        this.horaAprehension = horaAprehension;
    }

    public boolean isEntendidos() {
        return entendidos;
    }

    public void setEntendidos(boolean entendidos) {
        this.entendidos = entendidos;
    }

    public String getMotivo_sinfirma() {
        return motivo_sinfirma;
    }

    public void setMotivo_sinfirma(String motivo_sinfirma) {
        this.motivo_sinfirma = motivo_sinfirma;
    }

    public String getRasgos() {
        return rasgos;
    }

    public void setRasgos(String rasgos) {
        this.rasgos = rasgos;
    }

    public String getVestimenta() {
        return vestimenta;
    }

    public void setVestimenta(String vestimenta) {
        this.vestimenta = vestimenta;
    }
    
    
    
    
    
    
}

File: SGK9/web/js/Imputado.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

function Imputado(cedula,nombre,apellido1,apellido2,sexo,edad,nacionalidad,nacimiento,informe,acta,direccion,telefono,alias,aprehendido,horaAprehension,entendidos,motivo_sinfirma,rasgos,vestimenta) {
    this.Imputado(cedula,nombre,apellido1,apellido2,sexo,edad,nacionalidad,nacimiento,informe,acta,direccion,telefono,alias,aprehendido,horaAprehension,entendidos,motivo_sinfirma,rasgos,vestimenta);
  }
  
  Imputado.prototype={
        cedula: "",
	nombre: "",
	apellido1: "",
        apellido2: "",
        sexo: '',
        edad: 0,
        nacionalidad: "",
        nacimiento: "",
  	informe: "",
	acta: "",
	direccion: "",
        telefono: "",
        alias: "",
        aprehendido: 0,
        horaAprehension: "",
        entendidos: 0,
        motivo_sinfirma: "",
        rasgos: "",
        vestimenta: "",
	Imputado: function(cedula,nombre,apellido1,apellido2,sexo,edad,nacionalidad,nacimiento,informe,acta,direccion,telefono,alias,aprehendido,horaAprehension,entendidos,motivo_sinfirma,rasgos,vestimenta){
                this.cedula=cedula;
		this.nombre=nombre;
		this.apellido1=apellido1;
                this.apellido2=apellido2;
                this.sexo=sexo;
                this.edad=edad;
                this.nacionalidad=nacionalidad;
                this.nacimiento=nacimiento;
                this.informe=informe;
		this.acta=acta;
		this.direccion=direccion;
                this.telefono=telefono;
                this.alias=alias;
                this.aprehendido=aprehendido;
                this.horaAprehension=horaAprehension;
                this.entendidos=entendidos;
                this.motivo_sinfirma=motivo_sinfirma;
                this.rasgos=rasgos;
                this.vestimenta=vestimenta;
	},
	toString:function(){
	  return this.informe;
	}
  };
  
  Imputado.from= function(plain){
    var imputado = new Imputado(plain.cedula,plain.nombre,plain.apellido1,plain.apellido2,plain.sexo,plain.edad,plain.nacionalidad,plain.nacimiento,plain.informe,plain.acta,plain.direccion,plain.telefono,plain.alias,plain.aprehendido,plain.horaAprehension,plain.entendidos,plain.motivo_sinfirma,plain.rasgos,plain.vestimenta);
	return imputado;
  };
  
    Imputado.to= function(imputado){
    return {
        _class : 'Imputado',
        cedula : imputado.cedula,
        nombre : imputado.nombre,
	apellido1 : imputado.apellido1,
        apellido2: imputado.apellido2,
        sexo: imputado.sexo,
        edad: imputado.edad,
        nacionalidad: imputado.nacionalidad,
        nacimiento: imputado.nacimiento,
        informe : imputado.informe,
        acta : imputado.acta,
	direccion : imputado.direccion,
        telefono: imputado.telefono,
        alias: imputado.alias,
        aprehendido: imputado.aprehendido,
        horaAprehension: imputado.horaAprehension,
        entendidos: imputado.entendidos,
        motivo_sinfirma: imputado.motivo_sinfirma,
        rasgos: imputado.rasgos,
        vestimenta: imputado.vestimenta
    };	
  };
